bin/regioninfoDatEditor.ts

```diff
@@ -0,0 +1,50 @@
+import os from 'os';
+import fs from 'fs';
+import dns from 'dns';
+import util from 'util';
+
+const lookup = util.promisify(dns.lookup);
+
+export class regioninfoDatEditor {
+  regioninfoDat: string;
+  constructor() {
+    let home = os.homedir();
+    if (fs.existsSync(home + "/AppData/LocalLow/Innersloth/Among Us/regioninfo.dat"))
+      this.regioninfoDat = home + "/AppData/LocalLow/Innersloth/Among Us/regioninfo.dat";
+    else
+      throw new Error("regioninfo.dat not found!");
+  }
+
+  public async setServer(name: string, ip: string, port: number) {
```

Comment:
It looks like this file is pretty much a dupe of `lib/util/regioninfoDatEditor.ts`? Ideally a bin/executable would be able to be called from the command line, like `npm run update-server <name> <ip>` or something which will then call out to the utility function for the business logic, and then write to the file on disk